Backend/src/main/java/com/example/Backend/exceptions/InvalidInputException.java

```diff
@@ -0,0 +1,11 @@
+package com.example.Backend.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.UNAUTHORIZED)
```

Comment:
This is a nice way. you can remove the http status from the controller advice and test it

Reply:
You can remove the redundant Http Status from here if you face errors to remove from Controller advice